lib/kv.js

```diff
@@ -1,12 +1,35 @@
+const fs = require('fs')
+const path = require('path')
 const streams = require('@mattiasbuelens/web-streams-polyfill/ponyfill')
 
 class KeyValueStore {
-  constructor () {
+  constructor (name) {
```

Comment:
Rather than `name`, let's call this `path` and have it be the absolute or relative path to the file. This will simplify the logic below. We will leave it to the caller to decide on the extension and where the file get's stored.

Reply:
Additionally, if `path` has a length of zero, is undefined or null persistence should be considered disabled.

Reply:
I've updated to use a path now